src/server/console/logind_console_services.cpp

```diff
@@ -104,9 +107,9 @@ std::unique_ptr<LogindSession, decltype(&g_object_unref)> simple_proxy_on_system
 
                 auto error_msg = error ? error->message : "unknown error";
                 BOOST_THROW_EXCEPTION((
-                    std::runtime_error{
-                        "Failed to connect to DBus interface at "s +
-                        object_path + ": " + error_msg}));
+                                          std::runtime_error{
+                                              "Failed to connect to DBus interface at "s +
+                                              object_path + ": " + error_msg}));
```

Comment:
There are a lot of spurious, whitespace changes to this file. A lot of them away from project style